Some docs updates (#902)
zenml-io/zenml: docs/book/mlops-stacks/orchestrators/custom.md

## Direct Orchestration

> The implementation of a `local` orchestrator can be summarized in  lines of

Reviewer: ```suggestion
The implementation of a `local` orchestrator can be summarized as follows:
```
